battleship-client/src/main/java/com/academy/solid/nie/client/ui/ShipPlacer.java

```diff
@@ -5,26 +5,28 @@
 import javafx.scene.input.MouseButton;
 import javafx.scene.input.MouseEvent;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+import java.util.Random;
 import java.util.stream.IntStream;
 
-
-/**
- *
- */
 class ShipPlacer {
+    private static final String SHIPS_SEPARATOR = ",\\|";
+    private static final String COORDINATES_SEPARATOR = ",";
     private static final int FOUR_MAST = 4;
```

Comment:
I'd recommend having an Enum for a fleet instead. You can then just iterate over its values and you will know when it's empty fairly easily.